dietpi/dietpi-software

```diff
@@ -12759,6 +12806,41 @@ _EOF_
 
 		fi
 
+		software_id=191 # snapcast-client
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+			# Client IP needs to be added to allowed IP list
+			local snapcast_server_ip=
+			local snapcast_server_port=1704
+			local invalid_entry=
+			while :
+			do
+				if G_WHIP_INPUTBOX "${invalid_entry}Please enter the IP address of your snapcast server machine" && [[ $G_WHIP_RETURNED_VALUE =~ ^[0-9.]+$ ]]; then
+					snapcast_server_ip=$G_WHIP_RETURNED_VALUE
+					break
+				else
+					invalid_entry='[ERROR] Please enter a valid IP address. Please retry...\n\n'
+				fi
+			done
+
+			while :
+			do
+				G_WHIP_DEFAULT_ITEM=$snapcast_server_port
+				if G_WHIP_INPUTBOX "${invalid_text}Please enter the Port of your snapcast server machine\n - default: 1704" && disable_error=1 G_CHECK_VALIDINT "$G_WHIP_RETURNED_VALUE" 1
+				then
+					snapcast_server_port=$G_WHIP_RETURNED_VALUE
+					break
+				else
+					invalid_text='[ERROR] Please enter a valid port number. Please retry...\n\n'
+				fi
+			done
+
+			G_CONFIG_INJECT 'SNAPCLIENT_OPTS[[:blank:]=]' "SNAPCLIENT_OPTS=\"-h ${snapcast_server_ip} -p ${snapcast_server_port}\"" /etc/default/snapclient
+			# Restart snapclient so changes take effect.
+			G_EXEC systemctl restart snapclient
```

Comment:
We'll add it to `dietpi-services`, so it will be restarted automatically at the end of all installs.